Makefile

```diff
@@ -15,6 +15,7 @@ install:
 .PHONY: bundle
 bundle:
 	docker compose run --rm app bundle install
+	docker compose run --rm app_test bundle install
```

Comment:
@ManuSG8 I had forgotten to define the `bundle install` action for the `app_test` service. Now it is included